charts/go-boilerplate-ddd/templates/bootstrap-postgres.yaml

```
@@ -0,0 +1,131 @@
+  completions: 1
+  parallelism: 1
+  backoffLimit: 3
+  template:
+    spec:
+      restartPolicy: Never
+      initContainers:
+      - name: wait-for-dependencies
+        image: busybox:1.37
+        env:
+        - name: DB_HOST
+          value: {{ .Values.global.externalPostgresDefinitions.connection.host | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)